worlds/archipidle/docs/guide_fr.md

```diff
@@ -1,11 +1,11 @@
 # Guide de configuration d'ArchipIdle
 
 ## Rejoindre une partie MultiWorld
-1. Générez un fichier `.yaml` à partir de la [page des paramètres du lecteur ArchipIDLE](/games/ArchipIDLE/player-settings)
+1. Générez un fichier `.yaml` à partir de
+   la [page des paramètres du lecteur ArchipIDLE](/games/ArchipIDLE/player-options)
 2. Ouvrez le client ArchipIDLE dans votre navigateur Web en :
-     - Accédez au [Client ArchipIDLE](http://idle.multiworld.link)
-     - Téléchargez le client et exécutez-le localement à partir du
-       [Page des versions d'ArchipIDLE GitHub](https://github.com/ArchipelagoMW/archipidle/releases)
+    - Accédez au [Client ArchipIDLE](http://idle.multiworld.link)
+    - Téléchargez le client et exécutez-le localement à partir du [Page des versions d'ArchipIDLE GitHub](https://github.com/ArchipelagoMW/archipidle/releases)
```

Comment:
You split the previous line into two but not this one. I'm just confused as to why

Reply:
I just missed the first line split when going back and trying to correct PyCharm's forced formatting, it should be fixed now